index.php

```diff
@@ -36,11 +36,11 @@ function printForm()
   echo
   '<form method="get" action=""> 
    <p><label>Benutzer:&nbsp;<input type="text" name="user" value="' . getVar('user','').'"></label></p>
-   <p><label>Tag:&nbsp;<input type="text" name="day" value="'.getVar('day', date('d')).'" size="2" maxlength="2"></label>&nbsp;
-      <label>Monat:&nbsp;<input type="text" name="mon" value="'.getVar('mon', date('m')).'" size="2" maxlength="2" /></label>&nbsp;
-      <label>Jahr:&nbsp;<input type="text" name="year" value="'.getVar('year', date('Y')).'" size="4" maxlength="4" /></label>&nbsp;
-      <label>Uhrzeit:&nbsp;<input type="text" name="hour" value="'.getVar('hour', date('H')).'" size="2" maxlegth="2" />&nbsp;:&nbsp;
-                           <input type="text" name="min" value="'.getVar('min', date('i')).'" size="2" maxlength="2" /></label></p>
+   <p><label>Tag:&nbsp;<input type="number" name="day" value="'.getVar('day', date('d')).'" size="2" maxlength="2"></label>&nbsp;
+      <label>Monat:&nbsp;<input type="number" name="mon" value="'.getVar('mon', date('m')).'" size="2" maxlength="2" /></label>&nbsp;
+      <label>Jahr:&nbsp;<input type="number" name="year" value="'.getVar('year', date('Y')).'" size="4" maxlength="4" /></label>&nbsp;
+      <label>Uhrzeit:&nbsp;<input type="date" name="hour" value="'.getVar('hour', date('H')).'" size="2" maxlegth="2" />&nbsp;:&nbsp;
+                           <input type="date" name="min" value="'.getVar('min', date('i')).'" size="2" maxlength="2" /></label></p>
```

Comment:
Müsste das hier nicht auch jeweils `number` sein?

Reply:
Hab’s in Commit e9e6e43808a75d4861976ab3fc50ae38e7c11916 geändert.

Reply:
> 
> 
> Müsste das hier nicht auch jeweils `number` sein?

Nein, ist ja ein Datum! Weiß nicht, wie sich das auf den restlichen Code auswirken würde, müsste jedoch funktionieren...

Reply:
Nach meinem Verständnis würde das nur bei einem einzelnen input-Element für das ganze Datum funktionieren.  Das erfordert dann aber auch Code-Änderungen.